docs/src/pages/careers/lead-designer.md

```diff
@@ -65,26 +68,42 @@ As a team leader, you will:
-- **Strong written and verbal communication skills**. As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
-- **Management of a design team**. Whether formally or informally, you have shown that you are a design leader and can provide guidance and support to other designers.
+- **Expert-level UI knowledge**. You can craft great-looking designs, spotting tiny details most people wouldn't pay attention to.
+- **Expert-level UX knowledge**. You can build intuitive solutions, both high and low-level ones. You care about the entire user journey as well as an interaction with a specific component.
+- **Sharp written and verbal communication skills**. Communicating clearly is fundamental to create and manage high-quality products and teams. You'll interface with our team, community members, enterprise customers and also contribute to user documentation.
+- **Managerial abilities**. You have either managed a team before or are feeling ready to start now. You've shown that you're a design leader at heart and can provide the needed guidance and support to other designers.
+- **Comfortable around design systems**. You've worked on one before and have experienced the challenges of developing it.
+- **Enough development autonomy**. Our products are essentially code. Knowing how to navigate through code is key to understanding MUI products and collaborating with engineers. You know how to work with CSS (mainly CSS-in-JS), JSX, and Git.
 
 ### What would be nice if you had, but isn't required
 
-- **You've contributed to a design system before**. You have worked on a library of components in the past, maybe for internal usage in your company.
+- **Experience with open-source products**. You have seen the particular product dynamics when in an OSS environment and are knowledgable of the challenges.
 
 ## Benefits & Compensation
 
 Competitive compensation depending on the profile and location.
 We are ready to pay top market rates for a person that can clearly exceed the role's expectations.
 You can find the other perks & benefits on the [careers](https://mui.com/careers/#perks-amp-benefits) page.
 
+## Interview steps
+
+Here's a brief overview of how this role's process will be carried through:
+
+- **Introduction call** (20 to 25 min ⎯ with a designer): A short call to quickly introduce each other.
+- **Career deep dive** (40 to 60 min ⎯ with a designer): We'll go through your career a bit more in-depth and will be interested in understanding your trajectory up to the point where you've applied to this role.
+- **Comparable experience review** (40 to 60 min ⎯ with a designer, product manager or tech lead): You'll present one to three (up to you) of your prior experiences that showcase how capable you're of taking this role.
```

Comment:
```suggestion
- **Comparable experience review** (40 to 60 min ⎯ with a designer, product manager or tech lead): We'll deep dive into a presentation of relevant past experiences to understand your approach to problem-solving and design. 
```

Reply:
I think that we could make it clear that we will dive into the portfolio at this step.

Reply:
Gerda's suggestion is quite clear to me ⎯ it's not necessarily diving into each and every portfolio item but rather the ones the candidate believe showcase the best the skills we're looking for!